isis/src/docsys/build/DevStart.xsl

```diff
@@ -0,0 +1,299 @@
+        <meta name="description" content="About the Integrated Software for Imagers and Spectrometers (ISIS), created and managed by the USGS Astrogeology Research Program. ISIS provides a comprehensive, user-friendly, portable tool for processing, analyzing, and displaying remotely sensed image data."/>
+        <meta name="publisher" content="USGS - GD - Astrogeology Program"/>
+        <meta name="author" content="Deborah Lee Soltesz, webteam@astrogeology.usgs.gov"/>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+        <meta name="country" content="USA"/>
+        <meta name="state" content="AZ"/>
+        <meta name="county" content="Coconino"/>
+        <meta name="city" content="Flagstaff"/>
+        <meta name="zip" content="86001"/>
+
+        <link rel="stylesheet" href="../assets/styles/IsisStyleCommon.css"/>
+        <link rel="stylesheet" href="../assets/styles/main.css"/>
+        <link rel="stylesheet" href="../assets/styles/menu.css"/>
+        <link rel="stylesheet" media="print" href="../assets/styles/print.css"/>
+        <!-- Dynamic analytics insertion to prevent running on local URLs -->
+        <xsl:text>&#xa;</xsl:text>
+        <script type="text/javascript">
+          //<xsl:comment><![CDATA[
+          (function() {
+            var usgsAnalytics = document.createElement('script');
+            usgsAnalytics.type = 'text/javascript';
+            usgsAnalytics.async = true;
+            usgsAnalytics.src = 'http://www.usgs.gov/scripts/analytics/usgs-analytics.js';
+            if('http:' == document.location.protocol) {
+              var s = document.getElementsByTagName('script')[0];
+              s.parentNode.insertBefore(usgsAnalytics, s);
+            }
+          })();
+          ]]></xsl:comment>
+        <xsl:text>&#xa;</xsl:text>
+        </script>
+      </head>
+
+      <body>
+
+        <div class="isisMenu">
+         <xsl:call-template  name="writeMenu"/>
+        </div>
+
+        <div class="isisContent">
+
+        <a href="http://www.usgs.gov">
+        <img src="../assets/icons/littleVIS.gif" width="80" height="22" border="0" alt="USGS"/></a><br/>
+
+
+        <p style="margin-top:10px; padding-bottom:0px;">
+        ISIS Documentation</p>
+
+        <hr/>
+
+
+        <table width="100%" cellpadding="0" border="0" cellspacing="0">
+          <tr valign="top">
+            <td align="left">
+              <h1>
+                Contributor Getting Started 
+              </h1>
+              <p>
+                As Free and Open Source Software (FOSS), everyone is welcome to contribute to ISIS.
+                We define our contributors as the members of the community that help improve the
+                software in some way. Community members can contribute by reporting a bug, writing
```

Comment:
Not a huge fan of "in some way", especially since you end intro with "many other ways". Remove?